assets/scss/_vars.scss

```diff
@@ -5,6 +5,9 @@ $grey-1: #6c6c6c;
 $pricing-card1: #2bd8dd;
 $pricing-card2: #0368f4;
 $white-0: #ffffff;
+$red-0: #fc3832;
+$black: #000;
+$white: #fff;
```

Comment:
has it sense using vars for black and white while you can write the colors name directly?

Reply:
yes, want to reference all colors in variables in one central location → DRY i.e. Single Source of Truth
I could've written `$black: black` but I like using the hex codes

Let's say later instead of pure white you want to use a slight off white, then you can just adjust this variable. Same for black, let's say you want to implement a dimmed dark theme like on github then you don't want your black to be pure black